### DISCUSSION TITLE
Displaying two stimuli with keyboard-canvas-response

### DISCUSSION THREAD

**Discussion by Max-Lovell** [jspsych/jsPsych#1947] [Q&A] [answered]
Hi all,

Is it possible to display two stimuli with the keyboard-canvas-response plugin?

thanks!

**Comment by Max-Lovell** [comment, marked as answer]
not sure how to delete but I have figured out the issue,         
```js
stimulus: function drawRect(c){
  c.getContext('2d').fillRect(0, stim_y, squareWidth, squareWidth);
  c.getContext('2d').fillRect(500, stim_y, squareWidth, squareWidth)
},
```

**Reply by becky-gilbert** [reply]
Thanks for the update, and glad you figured this out! 
(FYI I edited your comment to format the code, so that it's easier for others to read 👍)

**Reply by kurokida** [reply]
It looks like the problem has already been solved, but my plugin may be useful.

# jspsych-psychophysics plugin
https://jspsychophysics.hes.kyushu-u.ac.jp/
